hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicOzoneFileSystem.java

```diff
@@ -759,6 +759,8 @@ public FileStatus getFileStatus(Path f) throws IOException {
       if (ex.getResult().equals(OMException.ResultCodes.KEY_NOT_FOUND)) {
         throw new FileNotFoundException("File not found. path:" + f);
       }
+      LOG.warn("GetFileStatus failed for path {}", f, ex);
```

Comment:
To be consistent with the signature for this interface, `catch (IOException ex)` instead of `catch (OMException ex)`. 

I am not sure if the underlying implementation is guaranteed to always throw `OMException`.  For other non `IOExceptions` exceptions, it might be better to map others as to `IOException` as well.

Reply:
Changed OME to IOE